Glamourer/Api/GlamourerIpc.cs

```diff
@@ -78,6 +80,11 @@ public partial class GlamourerIpc : IDisposable
 
         _stateChangedEvent.Subscribe(OnStateChanged, StateChanged.Priority.GlamourerIpc);
         _gPose.Subscribe(OnGPoseChanged, GPoseService.Priority.GlamourerIpc);
+
+        _applyByGuidAllProvider = new ActionProvider<Guid, string>(pi, LabelApplyByGuidAll, ApplyByGuidAll);
```

Comment:
Can you add some simple test methods for the new functions to IpcTester, too, please?